ddl/util/syncer.go

```diff
@@ -388,3 +411,109 @@ func (s *schemaVersionSyncer) OwnerCheckAllVersions(ctx context.Context, latestV
 		time.Sleep(checkVersInterval)
 	}
 }
+
+const (
+	opDefaultRetryCnt     = 10
+	failedGetTTLLimit     = 20
+	checkCleanJobInterval = 1 * time.Second
+	opDefaultTimeout      = 3 * time.Second
+	opRetryInterval       = 500 * time.Millisecond
+)
+
+// NeededCleanTTL is exported for testing.
+var NeededCleanTTL = int64(-60)
```

Comment:
Why setting this `-60`? I don't get the point here

Reply:
In fact, 0 is OK, but just set a negative value just in case.

Reply:
Seems like this variable is accessed in multi-thread, so please use atomic to protect it.

Reply:
It only read this variable in `doCleanExpirePaths`. And In tests, it's only set once before we call `doCleanExpirePaths`.